src/containers/PaymentContainer/PaymentContainer.tsx

```diff
@@ -0,0 +1,136 @@
+import { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router';
+import shallow from 'zustand/shallow';
+
+import styles from '#src/pages/User/User.module.scss';
+import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
+import Payment from '#components/Payment/Payment';
+import { getReceipt } from '#src/stores/AccountController';
+import { useAccountStore } from '#src/stores/AccountStore';
+import { getSubscriptionSwitches } from '#src/stores/CheckoutController';
+import { useCheckoutStore } from '#src/stores/CheckoutStore';
+import { useConfigStore } from '#src/stores/ConfigStore';
+import { addQueryParam } from '#src/utils/location';
+import useOffers from '#src/hooks/useOffers';
+import { useSubscriptionChange } from '#src/hooks/useSubscriptionChange';
+
+const PaymentContainer = () => {
+  const { accessModel } = useConfigStore(
+    (s) => ({
+      accessModel: s.accessModel,
+      favoritesList: s.config.features?.favoritesList,
+    }),
+    shallow,
+  );
+  const navigate = useNavigate();
+
+  const {
+    user: customer,
+    subscription: activeSubscription,
+    transactions,
+    activePayment,
+    pendingOffer,
+    loading,
+    canRenewSubscription,
+    canUpdatePaymentMethod,
+    canShowReceipts,
+  } = useAccountStore();
+
+  const [showAllTransactions, setShowAllTransactions] = useState(false);
+  const [isLoadingReceipt, setIsLoadingReceipt] = useState(false);
+  const [selectedOfferId, setSelectedOfferId] = useState<string | null>(activeSubscription?.accessFeeId ?? null);
+  const [isUpgradeOffer, setIsUpgradeOffer] = useState<boolean | undefined>(undefined);
+
+  const { offerSwitches } = useCheckoutStore();
+  const location = useLocation();
+
+  const handleUpgradeSubscriptionClick = async () => {
+    navigate(addQueryParam(location, 'u', 'upgrade-subscription'));
+  };
+
+  const handleShowReceiptClick = async (transactionId: string) => {
+    setIsLoadingReceipt(true);
+
+    try {
+      const receipt = await getReceipt(transactionId);
+
+      if (receipt) {
+        const newWindow = window.open('', `Receipt ${transactionId}`, '');
+        const htmlString = window.atob(receipt);
+
+        if (newWindow) {
+          newWindow.opener = null;
+          newWindow.document.write(htmlString);
+          newWindow.document.close();
+        }
+      }
+    } catch (error: unknown) {
+      throw new Error("Couldn't parse receipt. " + (error instanceof Error ? error.message : ''));
+    }
+
+    setIsLoadingReceipt(false);
+  };
+
+  useEffect(() => {
+    if (accessModel !== 'AVOD') {
```

Comment:
```suggestion
    if (accessModel === 'SVOD') {
```
The accessModel can be register only as well (AUTHVOD), in which case we don't need anything connected to subscriptions.